lib/evss/base_service.rb

```diff
@@ -1,4 +1,16 @@
 # frozen_string_literal: true
+class EVSSErrors < Faraday::Response::Middleware
+  def on_complete(env)
+    case env[:status]
+    when 200
+      resp = env.body
+      if resp['success'] && !resp['success']
+        raise Exception, resp['messages']
```

Comment:
@saneshark is there a preferred class to use here when raising errors from within an API client? I tried creating a new error class that inherited from StandardError but then the message was getting lost in the application controller exception handler and was replaced with the error class name